IdentifiabilityODE.mpl

```diff
@@ -277,10 +278,32 @@ IdentifiabilityODE := proc(system_ODEs, params_to_assess, {p := 0.99, count_solu
     z_aux, w_aux,
     op(sort(mu))
   ]:
+
+  ###########
+  non_id := map(x -> ParamToOuter(x, all_vars), [op({op(theta)} minus {op(theta_l)})]):
+  if use_weights then
+  	if infolevel > 0 then
+    		PrintHeader("Applying Weighted Ordering", output_targets[log]):
+    		LogText(sprintf("\t=> Applying Weighted Ordering"), ProgressBar):
+  	end if:
+	weight_subs, poly_system, poly_vars, original_et_hat := SubsByDepth(system_ODEs, [op(Et_hat), z_aux * Q_hat - 1], vars, non_id);

```

Comment:
I did not find `original_et_hat` used anywhere. Why do we return it?
Also, why does this function takes nonidentifiable functions as parameters?